src/commands/voucher.js

```diff
@@ -0,0 +1,61 @@
+const { OOPS_TEXT } = require('../messages')
+const path = require('path')
+const jsonfile = require('jsonfile')
+
+var vouchersFile = path.join(__dirname,'../vouchers.json')
+var vouchersFileBis = path.join(__dirname,'../vouchersBis.json')
+
+const ERROR_NOT_DM_MESSAGE = 'Carefull ! Yoou need to give your voucher code to the bot in a private message !'
+const INVALID_VOUCHER = 'Invalid voucher code'
+
+function voucher (message, dogecoinNode, voucherCode) {
+  var account = message.author.tag.replace('#', '')
+
+  if (message.channel.type === 'dm') {
+    var data = jsonfile.readFileSync(vouchersFile)
+
+    var validVoucher = data.vouchers.find(function (element) {
+      return element === voucherCode
+    })
+
+    console.log(validVoucher)
+
+    if (validVoucher) {
+      dogecoinNode.getBalance('', function (err, balance) {
+        if (err) {
+          console.log(err)
+          message.channel.send(OOPS_TEXT)
+          return
+        }
+
+        // We don't have enough funds...
+        if (balance - 30 <= 0) {
```

Comment:
Yeah good idea ! Different amount of the voucher is necessary because the PR difficulty can vary, doesn't it ?

Reply:
This PR is link to issue #32 so not yet for the PR but the code can be used for the PRs bounty by offering a voucher to people who are not yet on a discord server.

Reply:
I think we can do it later and leave the possibility tot the admin to specify the amount when creating the voucher. Having an admin panel for doing this would be nice...